packages/account/codegen.json

```diff
@@ -30,7 +30,8 @@
           "UtxoId": "string"
         },
         "useTypeImports": true,
-        "nonOptionalTypename": true,
+        "dedupeOperationSuffix": true,
```

Comment:
are there scenarios in the future where we would want these to be set to `false`?

Reply:
@maschad What flag specifically? `dedupeOperationSuffix` ? 

I don't see a case where we want to use this flag as false since [its purpose](https://the-guild.dev/graphql/codegen/plugins/typescript/typescript-apollo-next#dedupeoperationsuffix) is to remove the duplicated suffix from operation types

Reply:
I'm referring to both `dedupeOperationSuffix` and `skipTypename` but I just realize that [this flag](https://the-guild.dev/graphql/codegen/plugins/typescript/typescript-operations) is for gql-codegen, otherwise I would have said that these shouldn't be flags but the codegen itself should be modified. Given it's an external lib my comment is no longer relevant though.